modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
+
+You can configure bare-metal hardware events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Create a Git repository where you manage your custom site configuration data.
+
+[NOTE]
+====
+Multiple `HardwareEvent` resources are not permitted.
+====
+
+.Procedure
+
+. To configure the AMQ Interconnect Operator and the Bare Metal Event Relay Operator, add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file:
++
+[source,yaml]
+----
+# AMQ interconnect operator for fast events
+- fileName: AmqSubscriptionNS.yaml
+  policyName: "amq-sub-policy"
```

Comment:
Please rename policy to "subscriptions-policy".

The policy name should align with policy for other CRs that can be applied at the same time in ZTP doc, so the HW events related CRs can be included into the same CR. This is important because the number of policies affect the ZTP deployment times.